packages/styles/stories/Components/switch/with-text.example.html

```diff
@@ -1,42 +1,30 @@
 
 <div class="fd-form-group">
     <div class="fd-form-item">
-        <div class="fd-form-label" id="label3">With inline text (off)</div>
-        <label class="fd-switch">
-            <span class="fd-switch__text">Off</span>
+        <div class="fd-form-label" id="label3">Text Switch (off)</div>
+        <label class="fd-switch fd-switch--text">
             <span class="fd-switch__control">
                 <input class="fd-switch__input" type="checkbox" name="" aria-labelledby="label3" value="" id="y21Y677251">
```

Comment:
Why aria-labelledby and not just combination of `input[id]` and `label[for]`?

Reply:
That's actually a good point. I need to remove these as this text div was for the old markup, won't be needed in the new one.